en/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md

```diff
@@ -0,0 +1,275 @@
+
+Now the monitored hosts will need two components to be monitored:
+
+* the `centreon_linux_local.pl` plugin
+* the NRPE3 daemon
+
+#### Installation
+
+Follow the instructions below:
+
+```bash
+yum install http://yum.centreon.com/standard/20.04/el7/stable/noarch/RPMS/centreon-release-20.04-1.el7.centos.noarch.rpm
+yum install centreon-nrpe3-daemon.x86_64 centreon-plugin-Operatingsystems-Linux-Local.noarch
+```
+
+> **NB:** To avoid installing the Centreon Yum repo on all your monitored Linux servers, both `http://yum-1.centreon.com/standard/20.04/el7/stable/noarch/RPMS/centreon-plugin-Operatingsystems-Linux-Local-20200602-094050.el7.centos.noarch.rpm` and `http://yum-1.centreon.com/standard/20.04/el7/stable/x86_64/RPMS/centreon-nrpe3-daemon-3.2.1-8.el7.centos.x86_64.rpm` (current version at the time this document is written) **but this installation mode won't allow the packages to be updated with `yum update`**.
+
+#### NRPE configuration
+
+Then change the value of the `allowed_hosts` parameter in `/etc/nrpe/centreon-nrpe3.cfg`:
+
+```ini
+[...]
+# ALLOWED HOST ADDRESSES
+# This is an optional comma-delimited list of IP address or hostnames
+# that are allowed to talk to the NRPE daemon. Network addresses with a bit mask
+# (i.e. 192.168.1.0/24) are also supported. Hostname wildcards are not currently
+# supported.
+allowed_hosts=127.0.0.1,::1
+[...]
+```
+
+And restart the service:
+
+```bash
+systemctl restart centreon-nrpe3.service
+```
+
+## Centreon Configuration
+
+### Create a host using the appropriate template
+
+Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by the following table:
+
+Once the template has been applied, the following macros can be customized:
+
+| Name             | Mandatory | Description                                                                                 |
+| :--------------- | :-------: | ------------------------------------------------------------------------------------------- |
+| NRPECLIENT       |     X     | Name of the plugin to use to dialog with the NRPE3 daemon (default: `check_centreon_nrpe3`) |
+| NRPEPORT         |     X     | TCP port the NRPE3 daemon is listening to (default: 5666)                                   |
+| NRPETIMEOUT      |     X     | Command timeout (default: 5s)                                                               |
+| NRPEEXTRAOPTIONS |           | Any extra option (default: `-u` to return an `UNKNOWN` state in case of a connection issue) |
+
+## FAQ - troubleshooting
+
+If everything is fine, this command:
+
+```bash
+/usr/lib64/nagios/plugins/check_centreon_nrpe3 -H monitored_host_ip -p 5666
+```
+
+should produce this output:
+
+```text
+NRPE v3.2.1
+```
+
+But it can occur that everything is not fine... Let's detail the most common errors.
+
+### "Connection refused"
+
+If the output of the command is:
+
+```text
+connect to address monitored_host_ip port 5666: Connection refused
+```
```

Comment:
You can I think just add the complete output (actually text snipper) as a title. It will be easier for the reader to find it through navigation on the right